code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -214,7 +214,7 @@
 	fits_max_w_class = 3
 	max_combined_w_class = 200 //Doesn't matter what this is, so long as it's more or equal to storage_slots * plants.w_class
 	w_class = W_CLASS_TINY
-	can_only_hold = list("/obj/item/weapon/reagent_containers/food/snacks/grown","/obj/item/seeds","/obj/item/weapon/grown", "/obj/item/weapon/reagent_containers/food/snacks/meat", "/obj/item/weapon/reagent_containers/food/snacks/egg", "/obj/item/weapon/reagent_containers/food/snacks/honeycomb")
+	can_only_hold = list("/obj/item/weapon/reagent_containers/food/snacks/grown","/obj/item/seeds","/obj/item/weapon/grown", "/obj/item/weapon/reagent_containers/food/snacks/meat", "/obj/item/weapon/reagent_containers/food/snacks/egg", "/obj/item/weapon/reagent_containers/food/snacks/honeycomb", "/obj/item/weapon/shard")
```

Comment:
hmm

not sure how I feel about this, it's SO far out of flavor for plantbags and it's not like we let them hold sheets just because you can get sheets from botany -- really not liking the idea that janitors might be asking for plant bags because they're the most efficient way to hold shards due to 200 space

Reply:
maybe a leather sharps bag at the biogenerator would be cool? could hold other sharp things

Reply:
I'll make them on another PR then. I'll remove the bag thing on this one.

Reply:
Removed. @Kurfursten